### PR TITLE
demo-playground:  start-demo-cluster.sh & some factoring

### PR DESCRIPTION
**Pull request by deepfire**
Factors shell scripts a little and adds a whole-cluster start script.

### CODE REVIEW

**Comment by coot** [comment]
In `ouroboros-network` (as in `cardano-sl`) we follow the convention `username/branch-name`, e.g. `deepfire/shell-scripts-demo`

File: demo-playground/start-demo-cluster.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+set -xe
+
+cabal new-build demo-playground
+
+atexit() {
+        pkill demo-playground
+}
+trap atexit EXIT
+
+start-node 0 --real-pbft
```
Comment (coot) [inline demo-playground/start-demo-cluster.sh]: Could you make the `--real-pbft` to be configurable, it will be really useful to use the others as well.

**Comment by coot** [comment]
Could you drop a line about this script in the `ouroboros-consensus/demo-playground/README.md` file.

File: demo-playground/start-demo-cluster.sh (hunk 1 above)
Comment (deepfire) [reply demo-playground/start-demo-cluster.sh]: Done in https://github.com/input-output-hk/ouroboros-network/pull/644.

**Comment by deepfire** [comment]
@coot, all done in https://github.com/input-output-hk/ouroboros-network/pull/644.

**Pull request closed without merging**